src/main/resources/templates/payload/REFUSAL_RECEIVED-event.ftl

```diff
@@ -1,6 +1,16 @@
 "refusal" : {
 "type": "${refusalType}",
 "agentId": "${officerId}",
+
+<#--<#if (type == "HH" || "NC") && refusalType == "HARD_REFUSAL">-->
```

Comment:
is this temporary commented out?

Reply:
This will be uncommented when we start doing NC and have implemented encryption